Makefile

```diff
@@ -476,7 +476,7 @@ bundle: operator-sdk deployments ## Generate bundle manifests and metadata, then
 	# For some reason, the profile CM is not added automatically to the
 	# bundle, but in general bundles can contain CMs, so let's add it
 	# manually
-	python hack/extract-cm.py $(BUNDLE_OPERATOR_MANIFEST) configMap security-profiles-operator-profile > ./bundle/manifests/security-profiles-operator-profile_v1_configmap.yaml
+	#python hack/extract-cm.py $(BUNDLE_OPERATOR_MANIFEST) configMap security-profiles-operator-profile > ./bundle/manifests/security-profiles-operator-profile_v1_configmap.yaml
```

Comment:
can we just delete this line and the above comment?

Reply:
oops lol

Reply:
thanks, done